examples/seq2seq_attn/seq2seq_attn.py

```diff
@@ -72,20 +73,18 @@ def __init__(self, train_data):
 
```

Comment:
```
tx.modules.AttentionRNNDecoder(
  ...
  input_size=self.target_embedder.dim + config_model.decoder
              ['attention']['attention_layer_size'],
```
`input_size` is not fixed yet. Any problem here?

Reply:
Sorry for the late reply. I will fix #55 in this pull request as well.

Reply:
Since the interface is changed, does the `AttentionRNNDecoder` doc (about `input_size`) need to be updated as well?